Student/Student.xcodeproj/project.pbxproj

```diff
@@ -1720,17 +1720,15 @@
 			files = (
 			);
 			inputFileListPaths = (
+				"$(SRCROOT)/../scripts/carthage/input.xcfilelist",
 			);
 			inputPaths = (
-				"$(SRCROOT)/../Carthage/Build/iOS/PSPDFKit.framework",
-				"$(SRCROOT)/../Carthage/Build/iOS/PSPDFKitUI.framework",
 			);
 			name = "Carthage Copy Frameworks";
 			outputFileListPaths = (
+				"$(SRCROOT)/../scripts/carthage/output.xcfilelist",
```

Comment:
Does student need all the reactive stuff to get bundled in, since it doesn't link to it?